src/material-examples/sticky-header-overview/sticky-header-overview-example.css

```diff
@@ -0,0 +1,75 @@
+.sticky-header-example .example-centered {
+  text-align: left;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  height: 300px;
+  width: 500px;
+  overflow: auto;
+}
+
+.sticky-header-example button {
+    position: fixed;
+    margin: 16px;
+    z-index: 99;
+}
+
+.sticky-header-example * {
+    margin: 0;
+    padding: 0;
+}
+
+.sticky-header-example body {
+    font: 14px/1.5 Helvetica, Arial, sans-serif;
+    -webkit-appearance: none;
+    -moz-appearance: none;
+}
+
+.sticky-header-example h1 {
```

Comment:
The example shouldn't have any `h1` - `h6` since it could be loaded within any section of the surrounding document. Can alternatively use a `<header>` element.